src/main/java/seedu/address/logic/commands/EditCategory.java

```diff
@@ -16,7 +16,7 @@
  * Edits a category in the ExpenseTracker
  */
 public class EditCategory extends Command {
-    public static final String COMMAND_WORD = "ecat";
+    public static final String COMMAND_WORD = "edcat";
```

Comment:
Good job in keeping the edit command syntax consistent with the rest of the edit commands.